Resources/Audio/Misc/attributions.yml

```diff
@@ -1,4 +1,15 @@
-- files: ["tension_session.ogg"]
+- files: ["emergency_meeting.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/commit/97945e7d08d1457ffc27e46526a48c0453cc95e4"
+
+- files: ["beast.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Partially taken from TG station and made by maximej (freesound.org) and then mixed together, converted to mono and shortened by brainfood1183 (github)."
+  source: "https://github.com/tgstation/tgstation/commit/3e34eb9e3fec7ff55c2db11ba2e7d3b4156443fa"
+  source: "https://freesound.org/people/maximej/sounds/596530/"
+
+- files: ["tension_session.ogg, salvage.ogg"]
```

Comment:
?

Reply:
If i mixed the sound from 2 different sources what do i use for new sound? i plan to remove this sound anyway though.